release notes/v0.29.0.md

```diff
@@ -0,0 +1,133 @@
+
+Expect more information soon, but for a brief example, xk6 will work somewhat like this:
+
+```bash
+xk6 build v0.29.0 --with github.com/k6io/xk6-k8s --with github.com/k6io/xk6-sql@v0.1.1
+
+./k6 run some-script-with-sql-and-k8s.js
+```
+
+Thanks, @andremedeiros, for pushing us to add plugins in k6 and for making a valiant attempt to harness Go's poor plugin API! Thank you, @mardukbp, for pointing us towards xcaddy and explaining its benefits!
+
+### Support for setting local IPs, potentially from multiple NICs ([#1682](https://github.com/loadimpact/k6/pull/1682))
+
+You can now specify a list of source IPs, IP ranges and CIDRs for `k6 run`, from which VUs will make requests via the new `--local-ips` CLI flag or `K6_LOCAL_IPS` environment variable. The IPs will be sequentially given out to VUs, allowing you to distribute load between different local addresses. This option doesn't change anything on the OS level, so the IPs need to already be configured on the OS level in order for k6 to be able to use them.
+
+The biggest use case for this feature is splitting the network traffic from k6 between multiple network adapters, thus potentially greatly increasing the available network throughput. For example, if you have 2 NICs, you can run k6 with `--local-ips="<IP-from-first-NIC-1>,<IP-from-first-NIC-2>"` to balance the traffic equally between them - half of the VUs will use the first IP and the other half will use the second. This can scale to any number of NICs, and you can repeat some local IPs to give them more traffic. For example, `--local-ips="<IP1>,<IP2>,<IP3>,<IP3>"` will split VUs between 3 different source IPs in a 25%:25%:50% ratio.
+
+Thanks to @ofauchon, @srguglielmo, and @divfor for working on previous iterations of this!
+
+### New option for blocking hostnames ([#1666](https://github.com/loadimpact/k6/pull/1666))
+
+You can now block network traffic by hostnames with the new `--block-hostnames` CLI flag / `K6_BLOCK_HOSTNAMES` environment variable / `blockHostnames` JS/JSON option. Wildcards are also supported at the beginning, allowing you to easily block a domain and all of its subdomains. For example, this will make sure k6 never attempts to connect to any `k6.io` subdomain (`test.k6.io`, `test-api.k6.io`, etc.) and `www.example.com`:
+
+```js
+export let options = {
+  blockHostnames: ["*.k6.io" , "www.example.com"],
+};
+```
+
+Thanks to @krashanoff for implementing this feature!
+
+## UX and enhancements
+
+- HTTP: The [gjson library](https://github.com/tidwall/gjson/) k6 uses for handling the HTTP `Response.json(selector)` behavior was updated, so we now support more [modifiers](https://github.com/tidwall/gjson/blob/master/SYNTAX.md#modifiers) like `@flatten` and multipaths ([#1626](https://github.com/loadimpact/k6/pull/1626)). Thanks, @sondnm!
```

Comment:
We should probably mention this in the k6-docs PR.

Reply:
hmm the docs already forward to https://github.com/tidwall/gjson#path-syntax for the `.json()` selector option, so I am not sure we need to change anything